clippy_lints/src/large_enum_variant.rs

```diff
@@ -0,0 +1,81 @@
+//! lint when there are large variants on an enum
+
+use rustc::lint::*;
+use rustc::hir::*;
```

Comment:
Well, fortunately [you did not promise](https://github.com/Manishearth/rust-clippy/pull/1246#discussion_r81449854) :smile: